ArcadiaTeamsBot.ServiceDesk.Requests/RequestTypesMappingConfiguration.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ArcadiaTeamsBot.ServiceDesk.Requests
+{
+    using System.Collections.Generic;
+
+    public class RequestTypesMappingConfiguration : Dictionary<string, Dictionary<string, string>>
```

Comment:
You can specify that values in inner dictionary will be of type `RequestTypeForUIFieldType`.

Reply:
Sure. Fixed